dlt/destinations/bigquery/__init__.py

```diff
@@ -18,7 +18,7 @@ def _configure(config: BigQueryClientConfiguration = config.value) -> BigQueryCl
 def capabilities() -> DestinationCapabilitiesContext:
     caps = DestinationCapabilitiesContext()
     caps.preferred_loader_file_format = "jsonl"
-    caps.supported_loader_file_formats = ["jsonl", "sql"]
+    caps.supported_loader_file_formats = ["jsonl", "parquet"]
```

Comment:
removing `sql` fails the bigquery tests. `sql` is a valid file type that gets executed during loading and contains transformations